### PR TITLE
[NOP] link to _initialization from send_sms_example

### PR DESCRIPTION
**Pull request by Kirkkt**
@derrickstaten @dmitrig01 

this way we only have need to update one file per web sdk release.

kudos to Dmitri for pointing out the way for me :)

### CODE REVIEW

**Comment by Kirkkt** [comment]
BTW no visible diffs :)

File: ingredients/web_sdk/send_sms_example.md [hunk 1]
```diff
@@ -6,44 +6,47 @@ Here is a fully-functional web page that you can use as a template for your text
...
-    </head>
-    <body>
-    	Send SMS
-        <form onsubmit="sendSMS(this); return false;">
-        	<input id="phone" name="phone" type="tel" placeholder="(650) 123-4567" />
-        	<br/>
-        	<input type="submit"/>
-        </form>
-    </body>
+    {% ingredient web_sdk/_initialization %}
```
Comment (dmitrig01) [inline ingredients/web_sdk/send_sms_example.md]: hmm, the initialization ingredient already has the html highlight, so now it'll have highlighted html within highlighted html... maybe we shoudl add before_script and after_script sections to the ingredient, so you can override those with the relevant parts of this web page?

does that make sense?

cc @derrickstaten

**Comment by Kirkkt** [comment]
@dmitrig01 

I thought of something different - let's not change the current logic except adding a dedicated `_src.md` that is single-mindedly worrying about the right version to pull from.

Let me know what you think :)

**Comment by dmitrig01** [comment]
sure -t hat works too

**Comment by dmitrig01** [comment]
love it :+1:

**Comment by Kirkkt** [comment]
Unfortunately I have to drop this as there is some weird dependency issue I am unable to figure out and the time investment doesn't justify it.

**Pull request closed without merging**